### ISSUE TITLE
[Locale-CN] block value and block rating issue

### ISSUE THREAD

**Issue by clijiac** [pingzing/LittleBuster#19]
盾牌格挡值 = (block value)
盾牌格挡等级 = (block rating)

so 
#strings.zhCN.tl Line 62 shoule be
```
{ pattern = "使你的盾牌格挡等级提高(%d+)", location = "AfterValue" },
{ pattern = "使你的盾牌格挡等级提高(%d+点)", location = "AfterValue" },
```

not 
```
{ pattern = "使你的盾牌格挡值提高(%d+)", location = "AfterValue" },
{ pattern = "使你的盾牌格挡值提高(%d+点)", location = "AfterValue" },
```

**Comment by pingzing** [comment]
Oh, shoot. Thanks for the heads-up, I'll make sure to fix this in the next release.

**Comment by pingzing** [comment]
Fixed with #23. Will go out in the next release. If you could check that it's been fixed, that'd be fantastic!
